service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/ip/ControlledAzureIpResource.java

```diff
@@ -81,19 +85,40 @@ public ApiAzureIpResource toApiResource() {
 
   @Override
   public WsmResourceType getResourceType() {
-    return WsmResourceType.AZURE_IP;
+    return WsmResourceType.CONTROLLED_AZURE_IP;
```

Comment:
i'm a little confused why we have both wsmResourceType and wsmCloudResourceType when it is just cloudResourceType + stewardshipType

Reply:
I want to use the WsmResourceType enum to dispatch to the appropriate ResourceHandler. We have unique handlers for referenced and controlled resources. I have observed in the code we have a number of places matching up stewardship and resource type and having to test for unimplemented cases. By unifying the list we can get rid of that code.